crates/hir_def/src/attr.rs

```diff
@@ -32,9 +32,9 @@ impl Documentation {
     }
 }
 
-impl Into<String> for Documentation {
-    fn into(self) -> String {
-        self.0
+impl From<Documentation> for String {
+    fn from(Documentation(string): Documentation) -> Self {
```

Comment:
This is a little awkward. It took me a second to remember that this was valid rust.